### DISCUSSION TITLE
Is there a way to see if a Cask needs sudo permissions?

### DISCUSSION THREAD

**Discussion by mietzen** [Homebrew/discussions#3644] [Casks] [answered]
Is there any way to determine if a cask will require sudo permissions without installing it?

**Comment by SMillerDev** [comment, marked as answer]
Sometimes the developer documents if the installer needs administrator access. But since this is not governed by Homebrew there is no standard way.

**Comment by Kentzo** [comment]
Depends on the installer's type. Packages (pkg) can be inspected with a 3rd party tool (e.g. Suspicious Package). You're looking for the Distribution file that's embedded inside.

If installer is an app there is no way to tell, as it can do arbitrary things.
